src/transformers/models/donut/image_processing_donut.py

```diff
@@ -232,6 +233,7 @@ def thumbnail(
                 The channel dimension format of the input image. If not provided, it will be inferred.
         """
         input_height, input_width = get_image_size(image, channel_dim=input_data_format)
+        size = get_size_dict(size)
```

Comment:
not clear why we needed these changes, was this causing CI failure?

Reply:
these are utils for old processors that don't support the new image size format yet

we might as well add these here since (1) they help w/ backwards compatibility, (2) make the image-text-to-text pipeline easier to implement, & (3) they just revert to a no-op if `size` already follows the new image size format